src/Rel8/Schema/Context/Nullify.hs

```diff
@@ -81,28 +76,6 @@ class Interpretation context => Nullifiable context where
     -> Col context ('Spec labels necessity x)
 
 
-instance Nullifiable Aggregate where
```

Comment:
I also didn't think much about this, but nothing seems to break with this class removed (probably because I removed other stuff at the same time). Same point, if you want to keep this, I'm OK with that if you show me how.